pynamodb/connection/base.py

```diff
@@ -839,6 +839,7 @@ def update_item(self,
                     table_name,
                     hash_key,
                     range_key=None,
+                    actions=None,
```

Comment:
should we append this to the end instead of the middle for users that are using positional args?

Reply:
Honestly I think this is ok.
While we could do that here we can't do that with `Model.update` because of how legacy conditionals are represented. If a user is using positional args it will fail with a ValueError instead of fail quietly.